posframe.el

```diff
@@ -410,6 +422,8 @@ You can use `posframe-delete-all' to delete all posframes."
          (font-width (default-font-width))
          (font-height (with-current-buffer (window-buffer parent-window)
                         (posframe--get-font-height position)))
+         (parent-text-scale-mode-amount (with-current-buffer (window-buffer parent-window)
+                                          (and text-scale-mode text-scale-mode-amount)))
```

Comment:
We check for `text-scale-mode` because if it is `nil` then the current value of `text-scale-mode-amount` is not applied on the parent frame.